pkgs/tools/networking/userhosts/default.nix

```diff
@@ -0,0 +1,23 @@
+{lib, stdenv, fetchFromGitHub, pkg-config, ncurses, libnl }:
+
+stdenv.mkDerivation rec {
+  pname = "userhosts";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "bobvanderlinden";
+    repo = "hosts";
+    rev = "a91d2e93e8606073542a24f2dda6b9e62f2ad494";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Sorry for the inconvenience, I forgot to push. It's fixed now.